Author: Reiner Herrmann <reiner@reiner-h.de>
Description: Remove build user and date from version string to make build reproducible

--- libnss-ldap-265.orig/vers_string
+++ libnss-ldap-265/vers_string
@@ -32,17 +32,17 @@ if ($ARGV[0] eq "-f") {
 	print $PROJECT, $SEP, $VERSION, "\n";
 } elsif ($ARGV[0] eq "-l") {
 	print "static const char SGS_VERS[160] = \"";
-	print '@(#)LIBRARY:', $PROGRAM, '  PROJECT:', $PROJECT, $SEP, $VERSION, '  DEVELOPER:', $AUTHOR,  '  BUILT:', $DATE;
+	print '@(#)LIBRARY:', $PROGRAM, '  PROJECT:', $PROJECT, $SEP, $VERSION;
 	print "\";\n";
 } elsif ($ARGV[0] eq "-c") {
 	print "const char SGS_VERS[160] = \"";
-	print '@(#)PROGRAM:', $PROGRAM, '  PROJECT:', $PROJECT, $SEP, $VERSION, '  DEVELOPER:', $AUTHOR,  '  BUILT:', $DATE;
+	print '@(#)PROGRAM:', $PROGRAM, '  PROJECT:', $PROJECT, $SEP, $VERSION;
 	print "\";\n";
 	print "const char VERS_NUM[10] = \"$VERSION\";\n";
 } elsif ($ARGV[0] eq "-n") {
 	print "$VERSION\n";
 } elsif ($ARGV[0] eq "-B") {
-	print 'PROGRAM:', $PROGRAM, '  PROJECT:', $PROJECT, $SEP, $VERSION, '  DEVELOPER:', $AUTHOR,  '  BUILT:NO DATE SET (-B used)',"\n";
+	print 'PROGRAM:', $PROGRAM, '  PROJECT:', $PROJECT, $SEP, $VERSION,"\n";
 } elsif ($ARGV[0] eq "-v") {
 	unlink($OUTFILE);
 	open(OUTFILE,">$OUTFILE");
@@ -52,7 +52,7 @@ if ($ARGV[0] eq "-f") {
 		print OUTFILE "#define USE_SGS_VERS\n\n";
 	}
 	print OUTFILE "const char SGS_VERS[160] = \"";
-	print OUTFILE '@(#)PROGRAM:', $PROGRAM, '  PROJECT:', $PROJECT, $SEP, $VERSION, '  DEVELOPER:', $AUTHOR,  '  BUILT:', $DATE;
+	print OUTFILE '@(#)PROGRAM:', $PROGRAM, '  PROJECT:', $PROJECT, $SEP, $VERSION;
 	print OUTFILE "\";\n\n";
 	print OUTFILE "const char VERS_NUM[10] = \"$VERSION\";\n\n";
 	close(OUTFILE);
@@ -78,6 +78,6 @@ if ($ARGV[0] eq "-f") {
 	print STDERR "\tTo create a project distribution, type tarup\n";
 	print STDERR "\tTo commit a project, type commit [message]\n";
 } else {
-	print 'PROGRAM:', $PROGRAM, '  PROJECT:', $PROJECT, $SEP, $VERSION, '  DEVELOPER:', $AUTHOR,  '  BUILT:', $DATE;
+	print 'PROGRAM:', $PROGRAM, '  PROJECT:', $PROJECT, $SEP, $VERSION;
 	print "\n";
 }
